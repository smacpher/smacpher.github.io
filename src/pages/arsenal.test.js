import Arsenal from "./arsenal.js"
import React from "react"
import renderer from "react-test-renderer"

describe("Arsenal", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Arsenal />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
