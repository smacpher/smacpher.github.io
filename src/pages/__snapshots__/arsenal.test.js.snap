// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arsenal renders correctly 1`] = `
<div
  className="layout"
>
  <div
    className="center"
  >
    <p>
      Loading...
    </p>
  </div>
</div>
`;
